### DISCUSSION TITLE
手机浏览器为何无法播放视频？是手机上有什么限制吗？

### DISCUSSION THREAD

**Discussion by iemobiu** [AlistGo/alist#2550] [General]
通过 Alist 挂载了阿里云盘，在手机上通过浏览器 Safari、chrome 访问 Alist，无法播放视频，视频为 MP4 格式，本身在手机中打开是没问题的。
另外，电脑端，mkv 格式的视频通过 chrome、brave 浏览器播放视频是正常的，但firefox、safari是无法播放的。

希望有知道的朋友帮忙解决下，非常感谢。

![IMG_3254](https://user-images.githubusercontent.com/115678292/205194926-ef21e0f2-4a04-490c-902c-f873b97725d4.jpg)

**Comment by wenge110110** [comment]
跟你一样的问题 蹲一个大佬

**Reply by shuanglu1** [reply]
IOS系统本身并不支持mp4和mkv等格式的视频，所以想要播放出来可以尝试使用外置播放器或者转码成m3u8格式即可

**Comment by LongEarsRabbit** [comment]
android 手机播放失败
使用edge浏览器，会一直在重试中，播放进度与总进度也都是00:00。
小米浏览器，显示加载速度后***kb/s后，提示播放失败
PC端访问播放正常(火狐浏览器)
